app/models/participant_profile.rb

```diff
@@ -73,6 +73,20 @@ def approved?
   # delivery_partner
   delegate :delivery_partner, to: :latest_induction_record, allow_nil: true
 
+  def self.eligible_to_change_cohort_and_continue_training(in_cohort_start_year:)
+    billable_states = %w[eligible payable paid].freeze
+
+    joins(:participant_declarations, schedule: :cohort)
+      .where(cohorts: { start_year: in_cohort_start_year - Cohort::OPEN_COHORTS_COUNT })
+      .where("participant_declarations.state IN (?) AND declaration_type != ?", billable_states, "completed")
+      .where.not(id: ParticipantDeclaration.billable.for_declaration(:completed).pluck(:participant_profile_id))
```

Comment:
Why do we need this as well as thought the previous query is excluding billable completed declaration profiles 🤔

Reply:
Yeah its a bit hard to follow, but essentially we want profiles with billable declarations that are not completed and also have 0 billable declarations that are completed